tensorflow/contrib/lite/README.md

```diff
@@ -90,7 +90,7 @@ The demo is resizing each camera image frame to (224 width * 224 height) to matc
 
 Similar to the Android demo app, there's an iOS camera app that uses exactly the same model (224 * 224 quantized Mobilenet).
 
-Follow the instructions to build and run the demo app:
+This demo app requires a camera so it doesn't work with simulators. It's needed to be executed on a real iOS device. Follow the instructions to build and run the demo app:
```

Comment:
"It's needed to be" --> "It needs to be"

Reply:
Done. Thanks!